src/index.ts

```diff
@@ -11,7 +11,7 @@ import {
   toggler,
 } from "./blockdom";
 import { mainEventHandler } from "./component/handler";
-import { EventBus, whenReady, loadFile } from "./utils";
+import { EventBus, whenReady, loadFile, markup } from "./utils";
```

Comment:
maybe we should just export them, flat, like everything else. what do you think?